EDDiscovery/EliteDangerous/EDCommander.cs

```diff
@@ -316,6 +316,8 @@ public static void Update(List<EDCommander> cmdrlist, bool reload)
                 }
             }
 
+            #if false       //TBD what is this code for?  turned off due to it not being understood
```

Comment:
CommanderPaths.json was created in response to some people sharing their user DB between different computers and having different paths to their journals on those computers.